src/test/java/org/dinosaur/foodbowl/test/persister/PhotoTestPersister.java

```diff
@@ -0,0 +1,35 @@
+package org.dinosaur.foodbowl.test.persister;
+
+import lombok.RequiredArgsConstructor;
+import org.dinosaur.foodbowl.domain.photo.domain.Photo;
+import org.dinosaur.foodbowl.domain.photo.persistence.PhotoRepository;
+
+@RequiredArgsConstructor
+@Persister
+public class PhotoTestPersister {
+
+    private final PhotoRepository photoRepository;
+
+    public PhotoBuilder builder() {
+        return new PhotoBuilder();
+    }
+
+    public final class PhotoBuilder  {
+
+        private String path;
+
+        public PhotoBuilder path(String path) {
+            this.path = path;
+            return this;
+        }
+
+        public Photo save() {
+            return photoRepository.save(
+                    Photo.builder()
+                            .path(path == null ? String.valueOf(System.currentTimeMillis()) : path)
+                            .build()
+            );
+        }
```

Comment:
```suggestion
        public Photo save() {
            Photo photo = Photo.builder()
                            .path(path == null ? String.valueOf(System.currentTimeMillis()) : path)
                            .build()
            return photoRepository.save(photo);
        }
```
이렇게 해주는게 가독성이 더 좋더라구요 👍